frontend/src/components/common/header/header.module.scss

```diff
@@ -250,6 +265,58 @@
   }
 }
 
+.block-search-mobile {
+  @media screen and (max-width: 767px) {
+    position: fixed;
+    top: 0;
+    right: 0;
+    left: 0;
+    z-index: $header-search-mobile-z-index;
+    display: flex;
+    width: 100%;
```

Comment:
why do you need width: 100% if you have right: 0 and left: 0?

Reply:
@foxt451 because a little higher in the code on line 218, you can see class .block-search, that is permanent and it has width: 40%, so I just redefine this prop to display correctly